src/styles/blocks/stars.css

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  min-width: 96px;
```

Comment:
Ok, but as for me is more clear to set proper margins between stars, then hardcode container min-width. What if we'll decide to make stars a bit larger?

Reply:
This makes sense